airbyte-webapp/src/locales/en.json

```diff
@@ -269,17 +269,23 @@
   "sources.cancelled": "Cancelled",
   "sources.incomplete": "Retrying",
   "sources.partialSuccess": "Partial Success",
```

Comment:
This doesn't need to be done in this PR, but from my brief looking it seems like these `sources.<job_status>` messages aren't being used anywhere, so they could be removed